apps/server/src/modules/account/repo/micro-orm/account.repo.ts

```diff
@@ -108,12 +108,46 @@ export class AccountRepo {
 		await this.em.flush();
 	}
 
-	public async searchByUsernameExactMatch(username: string, skip = 0, limit = 1): Promise<Counted<Account[]>> {
-		return this.searchByUsername(username, skip, limit, true);
+	public async searchByUsernameExactMatch(username: string, offset = 0, limit = 1): Promise<Counted<Account[]>> {
+		const [entities, count] = await this.em.findAndCount(
+			this.entityName,
+			{
+				username,
+			},
+			{
+				offset,
+				limit,
+				orderBy: { username: 1 },
+			}
+		);
+		const accounts = AccountEntityToDoMapper.mapEntitiesToDos(entities);
+
+		return [accounts, count];
 	}
 
-	public async searchByUsernamePartialMatch(username: string, skip = 0, limit = 10): Promise<Counted<Account[]>> {
-		return this.searchByUsername(username, skip, limit, false);
+	/**
+	 * @deprecated causes slow query, do not use
+	 */
```

Comment:
Why is this function deprecated? We are using it in the Superhero Dashboard for partial user search.

Reply:
use full text search if you wanna do that, until then its deprecated